rendered_notebooks/02_basic_preprocessing_exercise_01_solution.ipynb

```diff
@@ -45,6 +45,14 @@
      "text": [
```

Comment:
<p>We should get rid of this warning at the beginning of each notebook ...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lesteve/scikit-learn-tutorial/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Maybe we should just be explicit and pass <code>cv=5</code>. That will both be more explicit / readable and silence the warning for old scikit-learn versions.</p><p> </p>